### PR TITLE
add periodicgw package

### PR DESCRIPTION
**Pull request by L-ZHOU-CN**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by L-ZHOU-CN** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks.

File: recipes/periodicgw/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+about:
+  home: https://gitlab.com/scottkalevhansen/periodic-gw
+  summary: Generates doubly-periodic random 2D Darcy flow fields with arbitrary mean flow direction
+  license: LGPL-3.0-or-later
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - scottkalevhansen
```
Comment (BastianZim) [inline recipes/periodicgw/meta.yaml:44]: Do they agree to be a maintainer?

File: recipes/periodicgw/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+test:
+  imports:
+    - periodicgw
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/scottkalevhansen/periodic-gw
```
Comment (BastianZim) [inline recipes/periodicgw/meta.yaml:37]: This URL gives me a 404, is it correct?

**Review by BastianZim — COMMENTED** [review]
Hi @L-ZHOU-CN thanks for your submission to conda-forge. Just two small things and then this should be ready to go!

File: recipes/periodicgw/meta.yaml (hunk 1 above)
Comment (L-ZHOU-CN) [reply recipes/periodicgw/meta.yaml:44]: Yes, they agree to be a maintainer.

File: recipes/periodicgw/meta.yaml (hunk 2 above)
Comment (L-ZHOU-CN) [reply recipes/periodicgw/meta.yaml:37]: The URL gives a private repository by far. We will make it public soon.

File: recipes/periodicgw/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/periodicgw/meta.yaml:44]: Ok but we'd need them to confirm. Just a thumbs up on this comment should be sufficient.

Comment (scottkalevhansen) [reply recipes/periodicgw/meta.yaml:44]: I agree to be a maintainer!

File: recipes/periodicgw/meta.yaml (hunk 2 above)
Comment (scottkalevhansen) [reply recipes/periodicgw/meta.yaml:37]: I just changed the visibility to public.

**Comment by BastianZim** [comment]
Let's give the review team a ping then for the final go ahead and this should be ready.

@conda-forge/help-python

**Comment by L-ZHOU-CN** [comment]
@conda-forge/help-python The recipe is ready for your review. Thanks.

**Pull request merged**